src/components/auth/Login.jsx

```diff
@@ -29,10 +39,15 @@ export const Login = () => {
       }
    }
 
+   useEffect(() => {
+         handleAlertMessage(message, 'login')
+   }, [message])
    return (
       <form onSubmit={handleSubmit(onSubmit)}>
          <div className="flex flex-col my-1">
-            {/* email */}
+            {alertMessage ? (
```

Comment:
this is good since '' is a falsy value in javascript, but as an enhancement, try creating a boolean state in the useAlert Custom Hook, something like 'showAlert'